onboarding.qmd

```diff
@@ -2,118 +2,127 @@
+- [ ] Join the [group Hackmd](https://hackmd.io/team/cct-datascience?nav=overview), used for a lot of group notetaking. 
+    - [ ] You will need to create a Hackmd account, and then someone from the group will need to add you.
+- [ ] Get added to Toggl, for time tracking
+- [ ] Log in to get edit access to the website. Navigate to [https://datascience.cct.arizona.edu/user](https://datascience.cct.arizona.edu/user). Don't log in using the obvious log in form; click on "Log in with NetAuth" (or similar) and log in using your NetID credentials.
+- [ ] Log in to [Stache](stache.arizona.edu), used for sharing passwords and other sensitive information within the university.
+- [ ] Get added to the group [Posit Connect](https://viz.datascience.arizona.edu/connect/)
+    - You'll have to request access and list a sponsor. This is Kristina.
+- [ ] Log in to the group [Quarto Pub](https://quartopub.com)
+
+### Office space
+
+- [ ] After getting your Cat Card, fill out the form at https://access.bio5.org/ to request access to BSRL
+    *   Click on request access for Data7
+    *   There is a cafe in first floor lobby with coffee and food for purchase.
+    
+### Publicity!
+
+- [ ] Send Kristina a headshot to use for your page on the website.
+- [ ] Write a short blog post (Renata's is [here](https://datascience.cct.arizona.edu/news/2023/08/renata-diaz-joins-cct-data-science-team) and Eric's is [here](https://datascience.cct.arizona.edu/news/2022/06/welcome-eric-scott)) announcing that you've joined the team.
+- [ ] Write and update your blurb on your page on the website.
+- [ ] Also send Headshot, Bio, Links to Matt Rahr (rahr\@ag.arizona.edu) in CCT
+- [ ] Make a Mastodon post.
+- [ ] (Optional) Update your social bios, etc.
+
+### Integrating into group workflows
+
+- [ ] Read about group expectations and general information [here](https://osf.io/tzmhp/wiki/General%20Information/)
+- [ ] Populate your calendar with the current group meeting schedule. Kristina will send you invites.
+- [ ]  Participate in daily standup meetings, [biweekly sprint planning meetings](https://hackmd.io/ih-qR2B6RJ221MJEFeEqcA), [weekly outreach hour](https://hackmd.io/fxo335XYQXGUpIm6c46TkQ), and [biweekly incubator meetings](https://github.com/orgs/cct-datascience/projects/7). Review [project management OSF page](https://osf.io/tzmhp/wiki/Project%20Management/) to prepare for these.
+- [ ] Set up your standing meeting with Kristina.
+- [ ] Begin learning about and using software and data related to projects.
+    - This can include reading about incubators (on our GitHub project board), funded projects (talk to Kristina and current team members), or trainings. 
+- [ ] Fill out biweekly time sheets every other Friday morning by 10am AZ on [UAccess](https://uaccess.arizona.edu/), which also has pay stubs, insurance info, etc. 
+    * If you are salaried, you only need to put in 1 hour of "PAW - Present at Work" for the pay period (doesn't matter which week).
```

Comment:
I thought we only needed to do Exception reporting - only reporting sick, vacation, etc